ampersand-select-view.js

```diff
@@ -291,6 +319,18 @@ SelectView.prototype.setMessage = function (message) {
     }
 };
 
+SelectView.prototype.clearMessage = function () {
+    var mContainer = this.el.querySelector('[data-hook~=message-container]');
+    var mText = this.el.querySelector('[data-hook~=message-text]');
```

Comment:
I think it makes sense to do that.  I'll save the references once in `render()` and update `setMessage` and `clearMessage` accordingly. It's a shame selectView doesn't extend view as it would be nice to use the cacheElements method here.